proposal.rst

```diff
@@ -168,27 +210,83 @@ Project Details
 
 **Describe any existing work and concepts on which your project is based.**
 
-FILLME
+My work will be based on the GSoC 2017 Project `Clickstream Mining for 
+Learning to Rank training data <https://trac.xapian.org/wiki/GSoC2017/LetorClickstream>`_. 
+
+**Phase 1: Implementing Click Models**
+
+The previous work has implemented the simplified DBN model for clickstream mining. 
+I plan to finish the EM algorithm in order to get a better performance.
+
+
+Besides DBN model, I will implement two more click models, the Dependent click Model 
+(DCM) and the User Browsing Model (UBM). Both two are also highly 
+effective to improve document ranking. 
+
+I will first work on an abstract class for all click models and then try to implement 
+each model according to relevant papers. 
+
+
+
+**Phase 2: Documentation**
+
+The second phase of my work will be a detailed documentation of each model I 
+implemented. It will provide the Omega users with a brief view of click models and
+pros & cons of each model. I will continue to work on the clickmodel.rst in the 
```

Comment:
This is user documentation, so the first kind I was talking about in a previous comment.